apps/schools/models/assessments.py

```diff
@@ -23,7 +23,8 @@ class Meta:
 class InstitutionAssessmentCohorts(BaseModel):
     school = models.ForeignKey('School', db_column='sid', primary_key=True)
     assessment = models.ForeignKey('Assessment', db_column='assid',
-                                   blank=True, null=True, on_delete=models.SET_NULL)
+                                   blank=True, null=True,
```

Comment:
@shivangi - some assessment changes seem to be going in on this branch, not sure why. Can you check if nothing is getting overwritten?